README.md

```diff
@@ -86,14 +86,18 @@ It's best to have at least 30GB of free space for Docker containers and images.
 
 ## Releasing a new version
 
-If you want to release a new version of YamUI, do the following to create a release branch:-
+If you want to release a new version of YamUI, do the following to create a release branch. Replace version_type with `major`, `minor`
+or `patch` as appropriate, based on [SemVer](https://semver.org):-
+
 
 ```sh
 git checkout -b awesome-release-branch
-npm version patch -m "Release version %s"
+npm version version_type -m "Release version %s"
```

Comment:
maybe wrap use backticks around \`version_type\` for emphasis?

Reply:
I tried that - but neither that nor underscores nor HTML works within a code block :-/ So I went with something that would fail if copy/pasted.